pages/apis/graphql_api.md.erb

```diff
@@ -55,6 +55,58 @@ curl https://graphql.buildkite.com/v1 \
 
 For documentation on the full list of fields and types, see the interactive documentation in the [GraphQL Explorer](#getting-started).
 
+## GraphQL IDs
+
+All node types have an `id` property, which is a global identifier encoding the object’s type and UUID. Buildkite’s GraphQL IDs are based on the [Relay Global Object ID](https://facebook.github.io/relay/graphql/objectidentification.htm).
+
+For example, this is how to find the GraphQL ID of an organization:
+
+```graphql
+query {
+  organization(slug: "my-org") {
+    id
+  }
+}
+```
+
+```json
+{
+  "data": {
+    "organization": {
+      "id": "T3JnYW5pemF0aW9uLS0tYTk4OTYxYjctYWRjMS00MWFhLTg3MjYtY2ZiMmM0NmU0MmUw"
+    }
+  }
+}
+```
+
+If you have a GraphQL ID, you can use it with the global `node` query. You’ll need to specify the type using an [Inline Fragment](https://graphql.org/learn/queries/#inline-fragments) to access type-specific properties.
```

Comment:
What does the global node query do? I was planning to suggest swapping the order of the sentence around, but realised I don't know what the node query is doing 😄Starting with 'if you have a graphql id' sounds a bit like there's some circumstances when there isn't an id available to you, as opposed to you just not having found it yet. 

Something like: "To find <result of performing this query>, use your graphql id with the global 'node' query".

Reply:
Oh yeah! What about:

> You can look up any GraphQL node using its `id` and the global `node` query. To return specific properties of the object, you’ll need to specify the object’s type using an [Inline Fragment](https://graphql.org/learn/queries/#inline-fragments).
>
> For example, …

(for context, you'll probably need to use the fragment syntax 99.999% of the time)

Reply:
Yeah that sounds great!